access/types/stdasync.py

```diff
@@ -362,7 +361,7 @@ def _acceptSubmission(request, course, exercise, post_url, sdir):
     '''
     uids = get_uid(request)
     attempt = int(request.GET.get("ordinal_number", 1))
-    container_flag = settings.CONTAINER_MODE and "container" in exercise
+    container_flag = "container" in exercise
 
     # Backup synchronous grading.
     if not container_flag and not settings.CELERY_BROKER:
```

Comment:
This if block for "backup synchronous grading" must be removed. The `runactions` function is only used with the old chroot sandbox. Therefore, this if block does not work with containers.

Reply:
Done!